addons/binding/org.openhab.binding.loxone/README.md

```diff
@@ -89,8 +89,8 @@ This binding creates channels for controls that are [used in Loxone's user inter
 |Radio|[Radio button 8x and 16x](https://www.loxone.com/enen/kb/radio-buttons/)|`Number`|`Decimal` (select output number 1-8/16 or 0 for all outputs off)<br>`OnOffType.OFF` (all outputs off)|
 |Switch | [Virtual inputs](https://www.loxone.com/enen/kb/virtual-inputs-outputs/) of switch type<br>[Push-button](https://www.loxone.com/enen/kb/push-button/) | `Switch` |`OnOffType.*`|
 |TextState|[State](https://www.loxone.com/enen/kb/state/)|`String`|none (read-only value)|
-|Timed Switch | [Virtual inputs](https://www.loxone.com/enen/kb/virtual-inputs-outputs/) of timed switch type<br>[Push-button](https://www.loxone.com/enen/kb/push-button/) | `Switch` |`OnOffType.*`|
-|Dimmer| [Virtual inputs](https://www.loxone.com/enen/kb/virtual-inputs-outputs/) of dimmer type<br>[Dimmer](https://www.loxone.com/enen/kb/dimmer/) | `Dimmer` |`OnOffType.*`<br>`Percent`|
+|TimedSwitch | [Stairwell light switch or Multifunction switch](https://www.loxone.com/enen/kb/stairwell-light-switch/ or https://www.loxone.com/enen/kb/multifunction-switch/) of timed switch type<br>[Push-button](https://www.loxone.com/enen/kb/push-button/) | `Switch` |`OnOffType.*`|
+|Dimmer| [Dimmer](https://www.loxone.com/enen/kb/dimmer/) of dimmer type<br>[Dimmer](https://www.loxone.com/enen/kb/dimmer/) | `Dimmer` |`OnOffType.*`<br>`Percent`|
```

Comment:
@ppieczul I have no clue yet how .items and .sitemap files are structured. Can you assist a little bit here?

Reply:
Items and sitemap files apparently are a key method of defining the configuration amongst openHAB users. I don't use it personally, but I also had to learn it a bit when merging the binding to add these examples. I see a value in that every feature that we have should be correctly configurable through these files.

I used the [items](http://docs.openhab.org/configuration/items.html) and [sitemaps](http://docs.openhab.org/configuration/sitemaps.html) guides which helped me a lot.  If you copy over the examples from the readme file you can use them as a starting point, because they are verified to work. The changes which you make to the files you can see choosing basic Ui. Before that you have to set the sitemap name in basic ui configuration using paper UI.

Of course you must change the ids to your configuration. Then just adding one line to each items and sitemap files per control should be fine.

Good luck!